README.md

```diff
@@ -27,21 +27,28 @@ APP_WALLET_PASSPHRASE
 ```
 #### 5. Run tests
 
+**Run Build Command**
```

Comment:
Why do we need this? It will build the image anyway.

Reply:
removed